lib/mutant/reporter/cli/printer/coverage_result.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Mutant
+  class Reporter
+    class CLI
+      class Printer
+        # Reporter for mutation coverage results
+        class CoverageResult < self
+          # Run report printer
+          #
+          # @return [undefined]
+          def run
+            visit(MutationResult, object.mutation_result)
```

Comment:
This is a pass through ATM, I could evision a verbose mode where we show covered mutations, and tell why they where killed.